subsys/usb/host/class/usbh_hub_mgr.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _USBH_HUB_MGR_H_
+#define _USBH_HUB_MGR_H_
+
+#include <zephyr/sys/slist.h>
+#include "usbh_hub.h"
+
+enum usbh_hub_app_status {
+	kHubRunIdle = 0,
+	kHubRunInvalid,
+	kHubRunWaitSetInterface,
+	kHubRunGetDescriptor7,
+	kHubRunGetDescriptor,
+	kHubRunSetPortPower,
+	kHubRunGetStatusDone,
+	kHubRunClearDone,
```

Comment:
Zephyr uses Linux Kernel coding style which does not use `kCamelCase`, but uses `snake_case` and `SNAKE_CASE` for global variables

https://docs.zephyrproject.org/latest/contribute/style/code.html